submodules/wizard/views/step-appRestrictions.html

```diff
@@ -0,0 +1,86 @@
+<div id="step_app_restrictions" class="wizard-step">
+{{#with i18n.accountsApp.wizard.steps.appRestrictions as |appRestrictionsI18n|}}
+	<div class="step-title">
+		{{appRestrictionsI18n.title}}
+	</div>
+	<div class="step-body">
+		<form id="form_app_restrictions" name="form_app_restrictions" action="javascript:void(0);">
+			<div class="section-body flex-row-container">
+				<div class="flex-item-2">
+					<div class="flex-row-container">
+						<div class="flex-item-1">
+							{{appRestrictionsI18n.accessLevel.description}}
+						</div>
+					</div>
+					<div class="flex-row-container">
+						<div class="flex-item-1 flex-container access-level-toggle">
+							<div id="access_level" class="btn-group">
+								<input type="radio" name="accessLevel" id="access_level_full" class="radio-button input-hidden" value="full"{{#compare @root.data.accessLevel "===" "full"}} checked{{/compare}}/>
```

Comment:
For `radio` and `checkbox` inputs, we prefer to associate labels implicitly (e.g. the `input` element must be within the content of the `label` element):
```js
<label>
    <input>
    <span>
        My radio label
    </span>
</label>
```
With this method, you don't need to use `id` and `for` attributes to link both elements together.

Reply:
In this case it allows to set the styles via CSS with the adjacent sibling combinator, without adding extra classes via jQuery, like so:

```css
.radio-button {
    &:checked + label.monster-button {
        background: $color-highlight;
        color: $woodsmoke;
    }
}
```

Can I leave it this way?

Reply:
Ah, I didn't realize it was grouped buttons. Yes, that works!